_docs/manage/policies.md

```diff
@@ -0,0 +1,118 @@
+---
+layout: guide
+title: StorageOS Docs - Policies
+anchor: manage
+module: manage/policies
+---
+
+# Policies
+
+Policies are a simple Attribute-Based Access Control records that are used to permission users (or groups) to namespaces.
+
+>**Note**: On initial startup there is only the storageos user and no policies are in place. If no policies exist, all users can access all namespaces.
+
+## Format
+
+The format for policy rules is similar to that of Kubernetes' ABAC syntax, defining acceptable actions within the system. If any of the policy entries match the
+action is allowed.
+
+>**Note**: Admin users are treated as super-users which can perform any action within the system, regardless of policies set.
+
+```json
+{"spec":{"user": "bob", "namespace": "legal"}}
+{"spec":{"group": "compliance", "namespace": "legal"}}
+```
+
+The rules above will allow any user with the username bob, or in the group compliance, to access the legal namespace.
```

Comment:
What does access mean? Create/update/remove volumes? Clarify that users can't modify namespaces, only admins can.